### DISCUSSION TITLE
Eye diagram contour export

### DISCUSSION THREAD

**Discussion by YoelBi** [ansys/pyaedt#5482] [Q&A]
Is there a method to export the contour of an eye diagram?
in the picture  - just the red curve
![image](https://github.com/user-attachments/assets/23f859e5-6d1a-4eed-b79e-8937a22c24bf)

**Comment by Samuelopez-ansys** [comment]
Hi @YoelBi ,

Please take a look to the API:

https://aedt.docs.pyansys.com/version/stable/API/visualization/_autosummary/ansys.aedt.core.visualization.report.eye.AMIConturEyeDiagram.html

https://aedt.docs.pyansys.com/version/stable/API/visualization/_autosummary/ansys.aedt.core.visualization.post.common.Reports.statistical_eye_contour.html

**Comment by YoelBi** [comment]
Thanks,
however I couldn't see a contour export.
